src/targets/gpu/include/migraphx/gpu/context.hpp

```diff
@@ -319,6 +339,7 @@ struct context
     bool measure_perf                 = false;
     shared<hip_event_ptr> start_event = nullptr;
     shared<hip_event_ptr> stop_event  = nullptr;
+    shared<hip_event_ptr> begin_event, finish_event;
```

Comment:
Initialize with `nullptr` ?

Reply:
Does it have to be? It's set on construction of the stream in the init list

Reply:
Change this to be more explicit I suppose on initialization.